### ISSUE TITLE
[Mediaflux] Deal with file names with spaces

### ISSUE THREAD

**Issue by tashrifbillah** [PREDICT-DPACC/lochness#21]
- [ x ]

```sh
checking 1000 assets (1~1000)...
Missing: file:/tmp/tmpx2xdvxgo/F6VVM__052281_2020-02-07 09-19-15.csv (source asset:/projects/proj-5070_prescient-1128.4.380/mflux_dummy_data/all_BWH_actigraphy/01236/GENEActiv/F6VVM__052281_2020-02-07 09-19-15.csv)
Missing: file:/tmp/tmpx2xdvxgo/2020-02-07 09-19-15.bin (source asset:/projects/proj-5070_prescient-1128.4.380/mflux_dummy_data/all_BWH_actigraphy/01236/GENEActiv/2020-02-07 09-19-15.bin)

                0 assets [passed]
                2 assets [missing]
                0 assets [content mismatch]
                2 assets [checked]

    Output file: /tmp/tmpx2xdvxgo/diff.csv
Listening socket closed!


Warning: prepending missing / to asset/namespace path: '09-19-15.bin'.

Invalid arguments: Asset or asset namespace: '/projects/proj-5070_prescient-1128.4.380/mflux_dummy_data/all_BWH_actigraphy/01236/GENEActiv/2020-02-07' does not exist.
```

**Comment by tashrifbillah** [comment]
TODO mediaflux cli existence check

**Comment by tashrifbillah** [comment]
- [ ] Loop over list in each pattern
- [ ] Encrypt if encrypt is true

**Comment by tashrifbillah** [comment]
Fixed in PR #22
